src/inc/public/AppxPackaging.hpp

```diff
@@ -1733,6 +1746,14 @@ MSIX_API HRESULT STDMETHODCALLTYPE PackPackage(
 
 #endif // MSIX_PACK
 
+MSIX_API HRESULT STDMETHODCALLTYPE PackBundle(

```

Comment:
>PackBundle [](start = 35, length = 10)

This should be inside the #ifdef MSIX_PACK

Reply:
Updated.